poetry/utils/helpers.py

```diff
@@ -141,3 +141,18 @@ def is_dir_writable(path: Path, create: bool = False) -> bool:
         return False
     else:
         return True
+
+
+def get_max_workers(desired_max_workers: Optional[int] = None):
```

Comment:
Given that this is only used in the one location and not unit-tested separately, I think I'd prefer to see this as a `@staticmethod` `def _get_max_workers` below the function it's used in.